docs/Spark/gems/_gem-requirements.mdx

```diff
@@ -0,0 +1,30 @@
+export default function Requirements(props) {
+  return (
+    <div>
+      <table style={{ width: "auto", tableLayout: "auto" }}>
+        <thead>
+          <tr>
+            <th>Required dependencies</th>
+            <th>
+              <a href="./release_notes/version_chart/">Minimum version</a>
```

Comment:
Maybe instead of linking on the text "Minimum version", we could write "Minimum version (View supported versions)"
... and link it to the parenthetical.

Reply:
It might make the column header longer than it needs to be.

Reply:
True. I think how it is is OK